ember/app/ui/components/cases-table/template.hbs

```diff
@@ -60,11 +50,22 @@
       </div>
     </div>
     <DynamicTable
-      @loading={{and this.fetchCases.isRunning (eq this.cases.length 0)}}
-      @data={{this.cases}}
+      @loading={{and this.caseQuery.isLoading}}
+      @data={{this.caseQuery.value}}
       @config={{this.dynamicTableConfig}}
-      @loadMoreData={{perform this.fetchCases this.pageInfo.endCursor}}
-      @hasNextPage={{this.pageInfo.hasNextPage}}
+      @loadMoreData={{perform this.fetchMoreCases}}
```

Comment:
In your component this is an `@action` i think. Also why do you not call `(fn this.caseQuery.fetchMore)` directly?

Reply:
perform is an accident, direct call doesnt work as fetchmore is just an async function